content/applications/hr/lunch/products.rst

```diff
@@ -0,0 +1,59 @@
+========
+Products
+========
+
+Odoo’s *Lunch* app does not come with any products pre-configured. The individual products being
+offered must first be configured before orders can be placed.
+
+First, navigate to :menuselection:`Lunch app --> Configuration: Products`. Next, click the
+:guilabel:`New` button in the top-left corner and a blank product form loads.
+
+Enter the following information on the form:
+
+- :guilabel:`Product Name`: enter the name for the product. This field is required.
```

Comment:
```suggestion
- :guilabel:`Product Name`: enter the name for the product. This field is **required**.
```
always suggest adding emphasis to 'required' mentions, just so it can help stand out a bit more.